cmd/config.go

```diff
@@ -161,6 +161,15 @@ type Config struct {
 
 	PA PAConfig
 
+	Statistics struct {
```

Comment:
Lets add this to `cmd/statistics/main.go` instead and use a separate config file, we are trying to stop adding any new fields to this struct.